numpy/random/bit_generator.pyx

```diff
@@ -294,6 +294,27 @@ cdef class SeedSequence():
     >>> sq2 = np.random.SeedSequence(sq1.entropy)
     >>> np.all(sq1.generate_state(10) == sq2.generate_state(10))
     True
+    
+    Examples
+    --------
+    
```

Comment:
Instead of repeating the examples with slightly different wording, I would probably just refer back to `SeedSequence`.

Reply:
As a user, I'd first look into the docstring. But am happy to change otherwise.